Lesson32/Merve/src/components/Exercise5.js

```diff
@@ -0,0 +1,36 @@
+import { useState, useEffect } from 'react';
+
+export const Exercise5 = ({ children }) => {
+  const [coordinates, setCoordinates] = useState({ x: 0, y: 0 });
+  const [tracking, setTracking] = useState(false);
+
+
+  const handleMouseMove = (e) => {
+    setCoordinates({
+      x: e.clientX,
+      y: e.clientY,
+    });
+  };
+
+  useEffect(() => {
+    if (tracking) {
+      window.addEventListener('mousemove', handleMouseMove);
+    } else {
+      window.removeEventListener('mousemove', handleMouseMove);
+    }
+
+    return () => {
+      window.removeEventListener('mousemove', handleMouseMove);
+    };
+  }, [tracking]);
+
+  const startTracking = () => setTracking(true);
+  const stopTracking = () => setTracking(false);
+
+  return children({
```

Comment:
This component should return a simple button and text element to display coordinates.. Once button is clicked, get the coordinates of the cursor and add them to the state.
You don't need to overcomplicate the solution and watch changes in the coordinates using useEffect.

Try to solve this exercises without using chatgpt or solution form internet as they are unnecessary complicated.